Sources/Orbit/Components/Timeline.swift

```diff
@@ -77,31 +80,31 @@ struct TimelinePreviews: PreviewProvider {
             TimelineStep(
                 "Booked",
                 sublabel: "January 3, 10:43",
-                style: .status(.success),
+                style: .pastSuccess,
                 description: "You booked the trip and received e-tickets."
             )
             TimelineStep(
                 "Checked in",
                 sublabel: "May 3, 8:45",
-                style: .status(.success),
+                style: .pastSuccess,
                 description: "You checked in for the trip and received boarding passes"
             )
             TimelineStep(
                 "Board",
                 sublabel: "May 4, 8:15",
-                style: .status(.warning),
+                style: .pastWarning,
                 description: "Be at your departure gate at least 30 minutes before boarding."
             )
             TimelineStep(
                 "Board",
                 sublabel: "May 4, 8:15",
-                style: .status(.critical),
+                style: .pastCritical,
                 description: "Be at your departure gate at least 30 minutes before boarding."
             )
             contentPlaceholder
```

Comment:
Preview should be updated to use the custom content directly, without client having to use `TimelineStepPreference` (this type should probably not be public at all)

```
            TimelineStep(
                "Board",
                sublabel: "May 4, 8:15",
                style: .pastCritical,
                description: "Be at your departure gate at least 30 minutes before boarding."
            ) {
                contentPlaceholder
            }
```

Inits might need to be updated, as this combination seems to be missing. Ideally, `content` should be renamed to `footer` so that call site can use full combination of header + default content + footer.

Reply:
I am not sure, if it should be done this way. At the moment there is no case in design where we would need to place custom content in-between the header and the footer. In fact, there is no footer, there is rather a description text, so maybe header + description + content would be a better idea? But honestly, I think that keeping it the way it's done now, with a public `TimelineStepPreference` is better because the content passed to the timeline is not type restricted so the client can pass other views there anyways. And at the moment they have a mechanism to draw the timeline correctly even if they decide to shove something else in there. If we make it internal, any custom view will break the layout and there won't be any mechanism to fix it. What do you think?

Reply:
`header + description + content` sure, it is just that footer would be more explicit in where the content would be put, as there are multiple of them now.

> if they decide to shove something else in there

Sure, but this something will always be a part of some step. I would not put this burden on caller - the simplier the API is, the better.

Reply:
Okay, added `Footer` and another initialiser